### DISCUSSION TITLE
Any chance to support 1+N interphone?

### DISCUSSION THREAD

**Discussion by dankocrnkovic** [PricelessToolkit/ESPBell-LITE#14] [General]
Hello,
 
Its hard for me to accept that I cant use this great product. So I must ask. Do you have any Idea what can I do with this interphone type?

![20240505_222559](https://github.com/PricelessToolkit/ESPBell-LITE/assets/78725973/0cd53733-9511-4f5c-9aa7-1cea9aa9166c)

**Comment by PricelessToolkit** [comment]
Hi, You can connect the ESPBell relay outputs to the opener button and ESPBell optocoupler input to that buzzer thing  "if you know how". But first, check if the door opener button works when the handset is not picked up.

**Reply by dankocrnkovic** [reply]
I'm afraid that would be out of my league... I could do it with more instructions, but I dont expect that you waste time on it. If someone from community could explain it in more detail maybe I would try...

**Comment by PricelessToolkit** [comment]
So that I can help, first check

1. Check if the door opener button works "can open the door" when the handset is not picked up
2.  Where is the doorbell coming from?

**Reply by dankocrnkovic** [reply]
Thank you....

1. Yes, door is buzzing on button press with handset down
2. There are 2 doorbels on this intercom. one is overriden with wago (bigger one). that one is my apartment door bell replaced with reolink doorbell. Second doorbell is building door. That one I open with button.

**Reply by PricelessToolkit** [reply]
ok so connect like this

 ESPBell "BELL" input to the Second doorbell.
 ESPBell "LOCK" wires to button contacts.

I can't explain it more precisely, Sorry.
